imports/plugins/core/ui/client/components/list/list.js

```diff
@@ -3,12 +3,12 @@ import PropTypes from "prop-types";
 import classnames from "classnames";
 import { registerComponent } from "@reactioncommerce/reaction-components";
 
-const List = ({ children, isAdmin }) => {
+const List = ({ children, isAdmin, className }) => {
```

Comment:
Done. PR to docs opened here https://github.com/reactioncommerce/reaction-docs/pull/256